backend/hitas/views/email.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+from rest_framework.viewsets import ViewSet
+
+from hitas.services.email import (
+    send_confirmed_max_price_calculation_email,
+    send_regulation_letter_email,
+    send_unconfirmed_max_price_calculation_email,
+)
+from hitas.views.utils import UUIDField
+
+
+class ConfirmedMaxPriceCalculationEmailSerializer(Serializer):
+    calculation_id = UUIDField()
+    template_name = serializers.CharField(max_length=256)
+    recipients = serializers.ListField(child=serializers.EmailField(), min_length=1)
+
+
+class UnconfirmedMaxPriceCalculationEmailSerializer(Serializer):
+    apartment_id = UUIDField()
+    calculation_date = serializers.DateField()
+    additional_info = serializers.CharField(allow_blank=True)
+    template_name = serializers.CharField(max_length=256)
+    recipients = serializers.ListField(child=serializers.EmailField(), min_length=1)
```

Comment:
Do we need a 'date this was requested' field here too? Assuming we want to track the 'work performance' here also

Reply:
Not sure, will ask.